Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_fs.h

```diff
@@ -64,7 +64,7 @@ class CWII_IPC_HLE_Device_fs : public IWII_IPC_HLE_Device
 
 	// ~1/1000th of a second is too short and causes hangs in Wii Party
 	// Play it safe at 1/500th
```

Comment:
This comment is a bit out of context here. I think it fits better in the implementation.